definitions.js

```diff
@@ -14,6 +15,7 @@ export const UNIT_CUP = 'cup';
 export const UNITS = [
   UNIT_LITER,
   UNIT_KILOGRAMS,
+  UNIT_GRAMS,
```

Comment:
I would really prefer to not add extra units for "multiples" as they can be handled quite easily using formatting functions.
I might miss context though. Something else might have been agreed with @madsnedergaard ?

Reply:
Nope, but makes sense to me :)